models/twitter_test.go

```diff
@@ -0,0 +1,42 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTwitterUserList(t *testing.T) {
+	list := &TwitterUserList{}
```

Comment:
`list` is reassigned and modified in the individual test cases and depends on the order of the tests.
Declare `list` inside the individual cases